.github/workflows/run-bench.yml

```diff
@@ -32,36 +32,90 @@ jobs:
     timeout-minutes: ${{ inputs.timeout }}
     steps:
       - uses: actions/checkout@v4
+        with:
+          submodules: 'recursive'
       - uses: actions/setup-node@v4
         with:
           node-version-file: 'package.json'
           cache: 'npm'
           cache-dependency-path: '**/package-lock.json'
+
+      # Setup pnpm
+      - name: Install pnpm
+        uses: pnpm/action-setup@v2
```

Comment:
Can we update this to latest, same with the cache one

Reply:
Definitely